app/views/layouts/_table_layout.html.erb

```diff
@@ -0,0 +1,26 @@
+
+<main class="grid_container">
+<section>
+  <ul>
+    <li>Trip ID</li>
+    <li>Date</li>
+    <li>Rating</li>
+    <li>Cost</li>
+    <li>Driver</li>
+  </ul>
+</section>
+
+<% @trips.each do |trip| %>
+
+<% if trip.passenger_id == @passenger.id %>
```

Comment:
Watch out for indentation  issues here - these are all nested code tags so you'll want to indent every step of the way